sql/_function.sql

```diff
@@ -76,6 +76,26 @@ CREATE OR REPLACE FUNCTION get_boro(
 
 CREATE OR REPLACE FUNCTION get_csd(
     _geom geometry
+  ) 
+    RETURNS varchar AS $$
+      SELECT lpad(b.schooldist::text,2,'0')::varchar
+      FROM dcp_school_districts b
```

Comment:
Why are we using the dcp_school_districts here?

Reply:
this was an error from earlier where the school community districts were confused with city council districts. Check the definition [here](https://github.com/NYCPlanning/db-developments/wiki/Data-Dictionary#schoolcommnty) for clarification